src/components/ui/Header.jsx

```diff
@@ -1,7 +1,11 @@
 function Header() {
   return (
     <header>
-      <nav>Header</nav>
+      <div className="flex items-start">
+        <h5 className="text-[var(--h5-size)]">
```

Comment:
Rather than using this approach we can define the text sizes for tailwind see [here](https://tailwindcss.com/docs/font-size#customizing-your-theme). 
```suggestion
        <h5 className="text-h5">
```

So we could add something like this to the @themes in the css and make use of tailwind for this (adjust as needed)
```css
  --text-h1: 3rem;           /* 48px */
  --text-h1--line-height: 3.5rem;
  --text-h1--font-weight: 700;

  --text-h2: 2rem;           /* 32px */
  --text-h2--line-height: 2.5rem;
  --text-h2--font-weight: 700;

  --text-h3: 1.5rem;         /* 24px */
  --text-h3--line-height: 2rem;
  --text-h3--font-weight: 600;

  --text-h4: 1.3rem;         /* 20.8px */
  --text-h4--line-height: 1.75rem;
  --text-h4--font-weight: 600;

  --text-h5: 1.1rem;         /* 17.6px */
  --text-h5--line-height: 1.5rem;
  --text-h5--font-weight: 500;

  --text-body: 1rem;         /* 16px */
  --text-body--line-height: 1.5rem;
  --text-body--font-weight: 400;

  --text-caption: 0.75rem;   /* 12px */
  --text-caption--line-height: 1rem;
  --text-caption--font-weight: 400;
 ```